mesh/education/urls.py

```diff
@@ -3,6 +3,11 @@
 from . import views as education_views
 
 urlpatterns = [
-    # POST  /education/
-    path('', education_views.EducationView.as_view(), name="education")
+    # POST  /educations/
+    # GET   /educations/
+    path('', education_views.EducationView.as_view(), name="education"),
+
+    # GET   /educations/:account_id
```

Comment:
Extremely small nitpick, but I believe I was just helping Brad with this issue, but clarify in this comment that the endpoint is /educations/:account_id/ with a trailing slash, as we were having problems where making requests to an endpoint that required a slash WITHOUT a slash, would not work correctly. So I would keep the trailing slash, just update the comment to match. Possibly a similar situation with the other two endpoints at /educations/

Reply:
I literally had this problem the first time I tried using Postman for this project. I will fix.